.circleci/config.yml

```diff
@@ -486,17 +486,17 @@ jobs:
       - run:
           name: Run linters
           command: |
+            # Run eslint as a standalone command to generate the test report.
+            PRE_COMMIT_NO_CONCURRENCY=true DISABLE=eslint pipenv run pre-commit run --show-diff-on-failure --color=always --all-files
             make jslint
-            make pylint
 
       - store_test_results:
           path: frontend/test-reports
           when: always
 
       - run:
-          name: Run type checkers
+          name: Run type checkers # tsc is run by pre-commit, thus in this step we only run mypy
           command: |
-            make tstypecheck
             scripts/mypy --report
```

Comment:
nit: Maybe we also just move this to the `Run linters` step and remove `Run type checkers` step since we anyways already do the typescript type checking in that step.